model/jpa/src/main/resources/default-persistence.xml

```diff
@@ -19,7 +19,7 @@
              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              xsi:schemaLocation="https://jakarta.ee/xml/ns/persistence https://jakarta.ee/xml/ns/persistence/persistence_3_0.xsd"
              version="3.0">
-    <persistence-unit name="keycloak-default" transaction-type="RESOURCE_LOCAL">
+    <persistence-unit name="keycloak-default" transaction-type="JTA">
```

Comment:
Currently, the default strategy for 'transaction-type' should be` JTA` as even the property `transaction-jta-enabled` is `true` by default. 

When I kept `RESOURCE_LOCAL`, I was facing an issue related to the initialization of Entity Manager Factory and specified `SynchronizationType`. AFAIK, it should not be possible to use it with `RESOURCE_LOCAL` tx type.

It means, the `JTA` is enabled by default.

Reply:
I think @ahus1 came across some issues with RESOURCE_LOCAL on quarkus too.

Reply:
@mabartos - when reviewing the JPA Map storage implementation in Quarkus 2.x with the Quarkus team, we've got the feedback that JTA is the only supported way going forward, while RESOURCE_LOCAL might or might not work in the future. 

It now seems we've reached the point were only JTA is working. Let's then remove everything else related to RESOURCE_LOCAL, and go with JTA all the way.

Reply:
Ok, thanks for the info, guys.